R/collect.R

```diff
@@ -349,16 +376,33 @@ collect_metrics.tune_results <- function(x, summarize = TRUE, ...) {
 
 #' @export
 #' @rdname collect_predictions
-collect_metrics.tune_race <- function(x, summarize = TRUE, ...) {
+collect_metrics.tune_race <- function(x, summarize = TRUE, complete = FALSE, ...) {
+
   x <- dplyr::select(x, -.order)
   if (summarize) {
     res <- estimate_tune_results(x)
   } else {
     res <- collector(x, coll_col = ".metrics")
   }
+  if (!complete) {
+    final_configs <- race_subset(x)
+    res <- dplyr::inner_join(res, final_configs, by = ".config")
+  }
   res
 }
 
+# Only return configurations that were completely resampled
+race_subset <- function(x) {
```

Comment:
A name like `subset_finished_race()` or similar could capture the comment above about which subset exactly